docs/en/macros/export-args.md

```diff
@@ -0,0 +1,20 @@
+---
+comments: false
+description: Comprehensive table on using the export function for Ultralytics YOLO models with detailed argument specifications.
+keywords: machine learning, model export, ONNX, TorchScript, TensorFlow, Keras, optimization, quantization, dynamic input, TensorRT, CoreML, NMS, batch inference
+---
```

Comment:
Will the action include all of this on every page?

Reply:
Yes, I updated the action to ignore pages under the macros folder! We don't want meta here!